tests/Gedmo/Loggable/Fixture/Entity/Composite.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Gedmo\Tests\Loggable\Fixture\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity
+ * @Gedmo\Loggable
+ */
+class Composite
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    private $one;
```

Comment:
Please, add type hints for these properties and argument / return declarations in the methods.

Reply:
Can do properties as this is still set to support PHP < 7.4

Reply:
With type hints I mean `@var` docblocks, not type declarations.